change/@fluentui-react-accordion-da96054b-7136-42a7-ac6d-414fea0b6b65.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29499&commitId=dd01e253d250d6d79dcc55883e928c5e396c1748&env=prod&clientType=FLUENTUI)


  <details><summary><b>MessageBar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Intents - RTL.default.chromium.png|5481| Changed|

</details>


  <details><summary><b>Toast</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toast.Full Toast - High Contrast.Toast visible.chromium.png|26260| Changed|
|Toast.Full Toast - RTL.Toast visible.chromium.png|7318| Changed|
|Toast.Full Toast - Dark Mode.Toast visible.chromium.png|20624| Changed|
|Toast.Without Subtitle - RTL.Toast visible.chromium.png|6509| Changed|
|Toast.Full Toast Inverted - Dark Mode.Toast visible.chromium.png|8417| Changed|
|Toast.Full Toast Inverted.Toast visible.chromium.png|21003| Changed|
|Toast.Full Toast.Toast visible.chromium.png|7327| Changed|
|Toast.Full Toast Inverted - High Contrast.Toast visible.chromium.png|26542| Changed|
|Toast.Without Subtitle - High Contrast.Toast visible.chromium.png|25143| Changed|
|Toast.Without Subtitle Inverted - High Contrast.Toast visible.chromium.png|25425| Changed|
|Toast.Without Subtitle Inverted - Dark Mode.Toast visible.chromium.png|7455| Changed|
|Toast.Without Subtitle - Dark Mode.Toast visible.chromium.png|19884| Changed|
|Toast.Without Subtitle Inverted.Toast visible.chromium.png|20069| Changed|
|Toast.Without Subtitle.Toast visible.chromium.png|6507| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>